R/formatters_var_labels.R

```diff
@@ -38,34 +38,28 @@ col_labels <- function(x, fill = FALSE) {
   checkmate::assert_data_frame(x)
```

Comment:
`col_labels` will not work with `DataFrame`. We need to relax this assertion to everything which can have `attr(<column>, "label")`. It was first fixed by @kartikeyakirar and got reverted because of a different reason.
My point is that functions in this file should be more universal and be also applicable to `DataFrame`.

Reply:
The assertions were causing CI failures and so they were reverted to what can be found in `formatters`. Note that this is now also on `main`.

All right, we need `data.frame` and `DataFrame`. Anything else? Matrices?